www/src/views/shared/sidebar.js

```diff
@@ -2,8 +2,8 @@ import React from "react"
 import MdFilterList from "react-icons/lib/md/filter-list"
 import styles from "../shared/styles"
 
-export const SidebarContainer = ({ children }) => (
-  <div css={[styles.sidebarContainer, styles.sticky]}>{children}</div>
+export const SidebarContainer = ({ children, style }) => (
+  <div css={[styles.sidebarContainer, styles.sticky, style]}>{children}</div>
```

Comment:
@pranshuchittora, this `style` is actually passed to css prop on a div below (is it `glamor` BTW? I can see there's `gatsby-plugin-emotion` installed, but it looks like `glamor`), so it's as inline as 
https://github.com/gatsbyjs/gatsby/blob/abe65803b0239a2aa811f73e1bdf9c18b626bd2a/www/src/views/starter-library/filtered-starters.js#L107
few lines below.

I couldn't name it `css` as in https://github.com/gatsbyjs/gatsby/issues/6649, so I've used `style`, but now I think the prop should probably be named `overrideCSS`  (as in Button) for consistence.

Or do you mean I should add this to `styles.sidebarContainer`? It would affect other pages though.